lessonbuilder/tool/pom.xml

```diff
@@ -94,8 +94,12 @@
             <artifactId>samigo-services</artifactId>
         </dependency>
         <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>commons-lang</groupId>
```

Comment:
Should we remove this dependency?

EDIT: I saw this comment:
//TODO commons-lang3

Reply:
Lang2: StringEscapeUtils.escapeJavaScript()

https://stackoverflow.com/questions/29146341/alternative-to-using-stringescapeutils-escapejavascript-in-commons-;ang3